### DISCUSSION TITLE
How to show string in explorer?

### DISCUSSION THREAD

**Discussion by ThomasBrixLarsen** [SanderMertens/flecs#882] [Q&A] [answered]
I thought it would be something like this:

```c++
struct Test
{
	const char *test = "test";
};

world.component<Test>().member<const char *>("test");
```

But it only shows a single char in explorer and so far the char it has shown has never been one of 't', 'e', 's', or 't'.

I was using newest master (e55fdc2ca02f48a11ab3a915bbacf3d6a6ee6b1c) for the test.

**Comment by SanderMertens** [comment, marked as answer]
The reason this happens is that the pointer is removed from the component declaration. To register a string member, do:
```c
world.component<Test>().member(flecs::String, "test");
```
